config/template.py

```diff
@@ -0,0 +1,50 @@
+import os
+import binascii
+
+def render_template(template, destination, **kwargs):
+
+
+   here = os.path.abspath(os.path.dirname(__file__))
```

Comment:
nit pick: This could be a constant (HERE in uppercase) defined at the beginning of the file.